policyengine_us/parameters/gov/states/ar/tax/income/credits/personal/personal.yaml

```diff
@@ -0,0 +1,21 @@
+description: Arkansas provides a personal credit of this amount, based on filing status.
+metadata:
+  label: Arkansas personal credit
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: Arkansas 2022 Individual Income Tax Forms and Instructions, page 12 - LINE 7A.
+    href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=12
+  - title: 2022 AR1000F ARKANSAS INDIVIDUAL INCOME TAX RETURN Full Year Resident
+    href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_FullYearResidentIndividualIncomeTaxReturn.pdf#page=1
+  - title: Ark. Code § 26-51-501 (a)(2)(A)(i)(a)
+    href: https://casetext.com/statute/arkansas-code-of-1987/title-26-taxation/subtitle-5-state-taxes/chapter-51-income-taxes/subchapter-5-tax-credits-generally/section-26-51-501-personal-tax-credits
+  - title: 2021 AR1000F ARKANSAS INDIVIDUAL INCOME TAX RETURN Full Year Resident
+    href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2021_AR1000F_FullYearResidentIndividualIncomeTaxReturn.pdf#page=1
+
+HEAD_OF_HOUSEHOLD:
```

Comment:
we need all 5 filing statuses here:
Joint
Single
separate
Widow
Head of household

Reply:
the other should be 0